### ISSUE TITLE
Drop-in replacement for FsCheck's Gen combinators in some cases

### ISSUE THREAD

**Issue by moodmosaic** [hedgehogqa/dotnet-hedgehog#47]
Perhaps, it makes sense to have something similar with [`disorder-jack/src/Test/QuickCheck/Jack.hs`](https://github.com/ambiata/disorder.hs/blob/master/disorder-jack/src/Test/QuickCheck/Jack.hs) here, in `dotnet-jack`, that basically maps Jack `Gen` combinators [to FsCheck ones](https://github.com/fscheck/FsCheck/blob/master/src/FsCheck/Gen.fs).

**Comment by moodmosaic** [comment]
Even with a mapping between Hedgehog and FsCheck `Gen` combinators, the two libraries work in a different way, so I don't think it's worth spending time on this.

In Hedgehog, shrinking is baked into the `Gen` type, so you get it for free. I'm not entirely sure how easy it is to transform Hedgehog's `Gen` type into  FsCheck's and vice-versa.

**Comment by jystic** [comment]
I think it might be possible to turn an FsCheck Arbitrary in to a Hedgehog Gen, but not the other way around

**Comment by mausch** [comment]
Relevant? https://hackage.haskell.org/package/hedgehog-quickcheck

**Comment by moodmosaic** [comment]
Yes, it's the one for the Haskell version of Hedgehog. It was actually [released today](https://github.com/hedgehogqa/haskell-hedgehog/pull/103) 😃

**Comment by moodmosaic** [comment]
We probably need to close #89 first before doing something similar in the F# version.
